src/app/components/ATIAnalytics/amp/index.test.tsx

```diff
@@ -27,4 +28,69 @@ describe('Amp ATI Analytics', () => {
         ?.innerHTML,
     ).toMatch(atiBaseUrl);
   });
+
+  it('creates an AMP analytics container with required parameters when the reverbParams prop is provided', () => {
+    const mockReverbParams = {
+      params: {
+        env: 'test',
+        page: {
+          contentId: 'urn:bbc:optimo:asset:cly1jw4x585o',
+          contentType: 'article',
+          destination: 'WS_NEWS_LANGUAGES_TEST',
+          name: 'gahuza.articles.cly1jw4x585o.page',
+          producer: 'GAHUZA',
+          additionalProperties: {
+            app_name: 'news-gahuza',
+            app_type: 'amp',
+            content_language: 'rw',
+            product_platform: null,
+            // eslint-disable-next-line no-template-curly-in-string
```

Comment:
Since this is used in 2 places, is it worth just suppressing it for the entire file?

Reply:
Good shout. I'll apply the exclusion to the entire file.

Reply:
This is now in place [here](https://github.com/bbc/simorgh/pull/12761/files#diff-5b3231bed418a28565ae3f8b709e4b276cea91e201f1e39726f13014a7dc65dcR1).